### PR TITLE
Deduplicate SecuityTetraplet with marine-rs-sdk

### PR DESCRIPTION
**Pull request by mikevoronov**

### CODE REVIEW

File: crates/air-lib/polyplets/src/lib.rs [hunk 1]
```diff
@@ -25,6 +25,16 @@
     unreachable_patterns
 )]
 
+extern crate self as marine_rs_sdk;
```
Comment (folex) [inline crates/air-lib/polyplets/src/lib.rs:28]: Wow, why is that needed?

Comment (ValeryAntopol) [reply crates/air-lib/polyplets/src/lib.rs:28]: #[marine] generates code that uses marine_rs_sdk::internal, so it is needed to provide it without depending on marine_rs_sdk

Comment (folex) [reply crates/air-lib/polyplets/src/lib.rs:28]: I see. Clever trick! I think it deserves a comment.

File: crates/air-lib/polyplets/src/triplet.rs [hunk 2]
```diff
@@ -19,6 +19,8 @@ use serde::Serialize;
 
 /// ResolvedTriplet represents peer network location with all
 /// variables, literals and etc resolved into final string.
+/// This structure contains a subset of value that
+/// SecurityTetraplet consist of
```
Comment (ValeryAntopol) [inline crates/air-lib/polyplets/src/triplet.rs:23]: value -> values
consist -> consists

**Review by ValeryAntopol — APPROVED** [review]
